Intermittent test failure: TestTask.test_generate_reportlogs_creates_for_both_types_of_changelogs

https://app.circleci.com/pipelines/github/mozilla/experimenter/10370/workflows/01bf0ece-816f-4708-97b3-5cc74eb45b84/jobs/30594

```
=================================== FAILURES ===================================
____ TestTask.test_generate_reportlogs_creates_for_both_types_of_changelogs ____
[gw0] linux -- Python 3.9.6 /usr/local/bin/python3

self = <experimenter.reporting.tests.test_tasks.TestTask testMethod=test_generate_reportlogs_creates_for_both_types_of_changelogs>

    def test_generate_reportlogs_creates_for_both_types_of_changelogs(self):
        mock_response_data = []
        mock_response = mock.Mock()
        mock_response.json = mock.Mock()
        mock_response.json.return_value = mock_response_data
        mock_response.raise_for_status = mock.Mock()
        mock_response.raise_for_status.side_effect = None
        mock_response.status_code = 200
        self.mock_normandy_requests_get.return_value = mock_response
        ExperimentFactory.create_with_status(target_status=Experiment.STATUS_LIVE)
        NimbusExperimentFactory.create_with_lifecycle(
            NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE
        )
        self.assertEqual(ReportLog.objects.count(), 0)
        generate_reportlogs()
        self.assertEqual(
            ReportLog.objects.filter(experiment_type__startswith="Normandy").count(), 5
        )
        self.assertEqual(
            ReportLog.objects.filter(experiment_type__startswith="Nimbus").count(), 3
        )
        self.assertEqual(ReportLog.objects.count(), 8)
    
        ExperimentChangeLogFactory.create(changed_on=timezone.now())
    
        self.assertEqual(ExperimentChangeLog.objects.count(), 6)
        generate_reportlogs()
    
>       self.assertEqual(ReportLog.objects.count(), 9)
E       AssertionError: 8 != 9

experimenter/reporting/tests/test_tasks.py:204: AssertionError
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1505)

--- a/app/experimenter/reporting/tests/test_tasks.py
+++ b/app/experimenter/reporting/tests/test_tasks.py
@@ -182,7 +182,9 @@ class TestTask(MockNormandyMixin, TestCase):
         mock_response.raise_for_status.side_effect = None
         mock_response.status_code = 200
         self.mock_normandy_requests_get.return_value = mock_response
-        ExperimentFactory.create_with_status(target_status=Experiment.STATUS_LIVE)
+        experiment = ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_LIVE
+        )
         NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE
         )
@@ -196,13 +198,17 @@ class TestTask(MockNormandyMixin, TestCase):
         )
         self.assertEqual(ReportLog.objects.count(), 8)
 
-        ExperimentChangeLogFactory.create(changed_on=timezone.now())
+        ExperimentChangeLogFactory.create(
+            changed_on=timezone.now(),
+            experiment=experiment,
+            old_status=Experiment.STATUS_LIVE,
+            new_status=Experiment.STATUS_COMPLETE,
+        )
 
         self.assertEqual(ExperimentChangeLog.objects.count(), 6)
         generate_reportlogs()
 
-        # Disable until https://github.com/mozilla/experimenter/issues/5976 lands
-        # self.assertEqual(ReportLog.objects.count(), 9)
+        self.assertEqual(ReportLog.objects.count(), 9)
 
     def test_generate_reportlogs_creates_for_normandy_changes(self):
         mock_response_data = [

